cli/simulate_pixels.py

```diff
@@ -407,6 +417,18 @@ def save_results(event_times, results, i_trig, i_mod=-1, light_only=False):
         RangePush("load_induction_response")
         response = cp.load(response_file)
         RangePop()
+
+        RangePush("load_pixel_thresholds")
+        if pixel_thresholds_file is not None:
```

Comment:
Does this break if either `pixel_thresholds_file` or `pixel_gains_file`are not set? There doesn't seem to be a catch that sets `pixel_{thresholds,gains}_lut` to their default values if the files are not indicated.

Reply:
I've only tested the case without setting pixel_{thresholds,gains}_file, where we use the const in the detector yaml, so the answer to your question is yes